tests/sb-util.test.ts

```diff
@@ -0,0 +1,30 @@
+import { initialize, ScratchProject } from '../packages/core/index';
+import process from 'process';
+
+test('ScratchProject not initialized on empty options', () => {	
+	// Empty options object
+	try {
+		initialize({});
+	} catch (e) {
+		expect(e.message).toContain('Please provide one of the following options');
+	}
+});
+
+test('ScratchProject not initialized on multiple present options', () => {
+	// Multiple options object
+	try {
+		initialize({ 'file' : 'foo.sb3', 'cloudId': 12345 });
+	} catch (e) {
+		expect(e.message).toContain('Multiple options found.');
+	}
+});
+
+test('ScratchProject intialized', async () => {
+	// Test init on sb3 file
+	await expect(initialize({ 'file': `${process.cwd()}/tests/data/test.sb3` })).resolves.toBeInstanceOf(ScratchProject);
+	
+	// Test init on project.json
+	await expect(initialize({ 'file': `${process.cwd()}/tests/data/project.json`})).resolves.toBeInstanceOf(ScratchProject);
+
+	await expect(initialize({ 'cloudId': 319383115 })).resolves.toBeInstanceOf(ScratchProject);
```

Comment:
We can not depend on an external server for tests. We need to test against a local proxy.

Reply:
We should either mock fetch or have any fetch calls throw error during testing.